### PR TITLE
SDK-657: acp integration enhancement

### PR DESCRIPTION
**Pull request by vol4onok**

### CODE REVIEW

**Comment by matweew** [comment]
Can you please find all places where the word "pbc" or "PBC" are used and replace it with "app" or "App"

**Review by gechetspr — APPROVED** [review]

**Review by andrea-taini — COMMENTED** [review]
"composer.lock" shouldn't be in .gitignore ?

File: src/Extension/Tasks/AcpAppAsyncApiAddMessageSchemaTask.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ placeholders:
     optional: true
     configuration:
       name: "asyncapi-file"
-      description: "Path to save AsyncAPI file to"
+      description: "Path to save AsyncAPI file to [default: 'config/api/asyncapi/asyncapi.yml']"
```
Comment (stereomon) [inline src/Extension/Tasks/AcpAppAsyncApiAddMessageSchemaTask.yaml:61]: The path has been changed to `resources/api/asyncapi.yml`.

File: src/Extension/Tasks/AcpAppOpenApiCodeGenerateTask.yaml [hunk 2]
```diff
@@ -22,19 +22,19 @@ placeholders:
     optional: true
     configuration:
       name: "openapi-file"
-      description: "Namespace that should be used for the code builder. When set to Spryker code will be generated in the core modules"
+      description: "Path to AsyncAPI file [default: 'config/api/openapi/openapi.yml']"
```
Comment (stereomon) [inline src/Extension/Tasks/AcpAppOpenApiCodeGenerateTask.yaml:25]: The path has been changed to `resources/api/openapi.yml`.

**Pull request merged**